openage/convert/main.py

```diff
@@ -126,11 +126,16 @@ def flag(name):
 
     used_asset_path = data_dir.resolve_native_path().decode('utf-8')
     if used_asset_path not in prev_srcdirs:
-        with asset_locations_path.open("a") as file_obj:
-            if len(prev_srcdirs) > 0:
-                file_obj.write("\n")
+        try:
+            with asset_locations_path.open("a") as file_obj:
+                if len(prev_srcdirs) > 0:
+                    file_obj.write("\n")
 
-            file_obj.write(used_asset_path)
+                file_obj.write(used_asset_path)
+
+        except IOError:
+            # cache file cannot be accessed, skip writing
+            pass
```

Comment:
such things should produce a log message so one can debug it easier :)

Reply:
good idea ^^